test/nn/conv/test_supergat_conv.py

```diff
@@ -16,6 +17,10 @@ def test_supergat_conv(att_type):
     out = conv(x, edge_index)
     assert out.size() == (4, 64)
 
+    adj_t = SparseTensor.from_edge_index(edge_index, sparse_sizes=(4, 4)).t()
+    out2 = conv(x, adj_t)
+    assert out2.size() == (4, 64)
+
```

Comment:
```suggestion
assert torch.allclose(out, out2)
```
Can we also compare the output of `get_attention_loss` ?